src/content/en/2020/compression.md

```diff
@@ -4,20 +4,268 @@ part_number: IV
 chapter_number: 19
 title: Compression
 description: Compression chapter of the 2020 Web Almanac covering HTTP compression, algorithms, content types, 1st party and 3rd party compression and opportunities.
-authors: [mo271]
-reviewers: [jyrkialakuijala, sboukortt, veluca93, paulcalvano]
+authors: [mo271, veluca93, sboukortt, jyrkialakuijala]
+reviewers: [paulcalvano]
 analysts: [AbbyTsai]
 translators: []
-#mo271_bio: TODO
+jyrkialakuijala_bio: Jyrki Alakuijala works as a software engineer at Google Switzerland.
+sboukortt_bio: Sami Boukortt works as a software engineer at Google Switzerland.
+mo271_bio: Moritz Firsching works as a software engineer at Google Switzerland.
+veluca93_bio: Luca Versari works as a software engineer at Google Switzerland.
```

Comment:
This is a nice short tagline, but consider adding more context to help readers understand your qualifications for writing on this topic, for example a paragraph rather than a sentence.

Reply:
@jyrkialakuijala @sboukortt @mo271 @veluca93

Reply:
added longer biographies.